tests/args_names.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+import shlex
+from re import compile
+
+from pykickstart.sections import PackageSection, PreScriptSection, PreInstallScriptSection,\
+    PostScriptSection, OnErrorScriptSection, TracebackScriptSection
+from pykickstart.version import makeVersion
+
+VALID_KICKSTART_OPTION_PATTERN = r"--[a-z0-9\-]+"
+
+
+class ArgumentNamesStatic_TestCase(unittest.TestCase):
+
+    def setUp(self):
+        self._bad_options = []
+        self._reg_name_checker = compile(VALID_KICKSTART_OPTION_PATTERN)
+        self._handler = makeVersion()
+
+    def runTest(self):
+
+        self._check_commands()
+        self._check_sections()
+        self._report()
+
+    def _check_sections(self):
+
+        sections = {
```

Comment:
The section openers should be populated from the class .sectionOpen instead of hard-coded here. Sure, they're not likely to change, but that will mean less things to change when adding new ones to check.